swift/codegen/schema.yml

```diff
@@ -0,0 +1,997 @@
+
+BridgeFromObjCExpr:
+  _extends: ImplicitConversionExpr
+
+BridgeToObjCExpr:
+  _extends: ImplicitConversionExpr
+
+ClassMetatypeToObjectExpr:
+  _extends: ImplicitConversionExpr
+
+CollectionUpcastConversionExpr:
+  _extends: ImplicitConversionExpr
+
+ConditionalBridgeFromObjCExpr:
+  _extends: ImplicitConversionExpr
+
+CovariantFunctionConversionExpr:
+  _extends: ImplicitConversionExpr
+
+CovariantReturnConversionExpr:
+  _extends: ImplicitConversionExpr
+
+DerivedToBaseExpr:
+  _extends: ImplicitConversionExpr
+
+DestructureTupleExpr:
+  _extends: ImplicitConversionExpr
+
+DifferentiableFunctionExpr:
+  _extends: ImplicitConversionExpr
+
+DifferentiableFunctionExtractOriginalExpr:
+  _extends: ImplicitConversionExpr
+
+ErasureExpr:
+  _extends: ImplicitConversionExpr
+
+ExistentialMetatypeToObjectExpr:
+  _extends: ImplicitConversionExpr
+
+ForeignObjectConversionExpr:
+  _extends: ImplicitConversionExpr
+
+FunctionConversionExpr:
+  _extends: ImplicitConversionExpr
+
+InOutToPointerExpr:
+  _extends: ImplicitConversionExpr
+
+InjectIntoOptionalExpr:
+  _extends: ImplicitConversionExpr
+
+LinearFunctionExpr:
+  _extends: ImplicitConversionExpr
+
+LinearFunctionExtractOriginalExpr:
+  _extends: ImplicitConversionExpr
+
+LinearToDifferentiableFunctionExpr:
+  _extends: ImplicitConversionExpr
+
+LoadExpr:
+  _extends: ImplicitConversionExpr
+
+MetatypeConversionExpr:
+  _extends: ImplicitConversionExpr
+
+PointerToPointerExpr:
+  _extends: ImplicitConversionExpr
+
+ProtocolMetatypeToObjectExpr:
+  _extends: ImplicitConversionExpr
+
+StringToPointerExpr:
+  _extends: ImplicitConversionExpr
+
+UnderlyingToOpaqueExpr:
+  _extends: ImplicitConversionExpr
+
+UnevaluatedInstanceExpr:
+  _extends: ImplicitConversionExpr
+
+UnresolvedTypeConversionExpr:
+  _extends: ImplicitConversionExpr
+
+BuiltinLiteralExpr:
+  _extends: LiteralExpr
+
+InterpolatedStringLiteralExpr:
+  _extends: LiteralExpr
+  interpolation_expr: OpaqueValueExpr?
+  interpolation_count_expr: Expr?
+  literal_capacity_expr: Expr?
+  appending_expr: TapExpr?
+
+RegexLiteralExpr:
+  _extends: LiteralExpr
+
+NilLiteralExpr:
+  _extends: LiteralExpr
+
+ObjectLiteralExpr:
+  _extends: LiteralExpr
+
+DynamicLookupExpr:
+  _extends: LookupExpr
+
+MemberRefExpr:
+  _extends: LookupExpr
+  base_expr: Expr
+
+SubscriptExpr:
+  _extends:
+    - GenericContext
+    - LookupExpr
+  base_expr: Expr
+  arguments: Argument*
+
+OverloadedDeclRefExpr:
+  _extends: OverloadSetRefExpr
+
+DoCatchStmt:
+  _extends: LabeledStmt
+  body: Stmt
+  catches: CaseStmt*
+
+DoStmt:
+  _extends: LabeledStmt
+  body: BraceStmt
+
+ForEachStmt:
+  _extends: LabeledStmt
+  body: BraceStmt
+  where: Expr?
+
+LabeledConditionalStmt:
+  _extends: LabeledStmt
+  condition: StmtCondition
+
+StmtCondition:
+  _extends: AstNode
+  elements: ConditionElement*
+  _dir: stmt
+
+RepeatWhileStmt:
+  _extends: LabeledStmt
+  cond: Expr
+  body: Stmt
+
+SwitchStmt:
+  _extends: LabeledStmt
+  subject_expr: Expr
```

Comment:
I'd prefer to call this `expr` (which is what it's called in [C++](https://github.com/github/codeql/blob/main/cpp/ql/lib/semmle/code/cpp/stmts/Stmt.qll#L1488), [C#](https://github.com/github/codeql/blob/main/csharp/ql/lib/semmle/code/csharp/Stmt.qll#L161), [Java](https://github.com/github/codeql/blob/b433f08cef9038d60e2bdf50bfedaf112f35d3f6/java/ql/lib/semmle/code/java/Statement.qll#L397) and [JavaScript](https://github.com/github/codeql/blob/main/javascript/ql/lib/semmle/javascript/Stmt.qll#L617)).